config/application.rb

```diff
@@ -115,7 +115,7 @@ class Application < Rails::Application
     config.site_version = "3.0.0"
 
     # Set application host for mailer
-    config.action_mailer.default_url_options = { host: ENV['MAILER_HOST'] || "YOUR_APP_HOST" }
```

Comment:
I guess a weird thing here is that this option doesn't really get reflected on the config side, but I did see damian's comment so I guess just removing the env part is fine?